paasta_tools/instance/kubernetes.py

```diff
@@ -441,11 +449,10 @@ def kubernetes_status(
     active_shas = kubernetes_tools.get_active_shas_for_service(
         [app, *pod_list, *replicaset_list]
     )
-    kstatus["app_count"] = max(
-        len(active_shas["config_sha"]), len(active_shas["git_sha"])
```

Comment:
Hmm, is this not important for bounces that just change config? I guess we don't wait-for-deployment in those cases?